src/App.svelte

```diff
@@ -89,9 +89,9 @@
     <label>
       ATIP schema:
```

Comment:
We already label all of these things as schemas. So "ATIP Schema: Schema (housing)" feels redundant -- why not just "ATIP Schema: housing"?

Reply:
> We already label all of these things as schemas. So "ATIP Schema: Schema (housing)" feels redundant -- why not just "ATIP Schema: housing"?

Oh yeah, good point.

Reply:
Updated.